kuksa-client/README.md

```diff
@@ -55,36 +55,48 @@ Welcome to Kuksa Client version <some_version>
 
 Default tokens directory: /some/path/kuksa_certificates/jwt
 
-connectj to wss://127.0.0.1:8090
-Websocket connected securely.
+Connecting to VSS server at 127.0.0.1 port 55556 using KUKSA GRPC protocol.
+TLS will not be used.
+INFO 2023-09-15 18:48:13,415 kuksa_client.grpc No Root CA present, it will not be posible to use a secure connection!
+INFO 2023-09-15 18:48:13,415 kuksa_client.grpc.aio Establishing insecure channel
+gRPC channel connected.
```

Comment:
shouldn't it connect to websocket in this example?

Reply:
I reordered it know, according to Eriks comment: It first shows the GRPC output, and AFTERWARDs mentions the command to connect to WS (without giving any output). Should be less confusing